libs/application/templates/party-letter/src/fields/PartyLetter/index.tsx

```diff
@@ -36,7 +37,7 @@ const PartyLetter: FC<FieldBaseProps> = () => {
   const { formatMessage } = useLocale()
 
   const first = ACTIVE_PARTIES.slice(0, Math.ceil(ACTIVE_PARTIES.length / 2))
-  const second = ACTIVE_PARTIES.slice(Math.ceil(ACTIVE_PARTIES.length / 2) + 1)
+  const second = ACTIVE_PARTIES.slice(Math.ceil(ACTIVE_PARTIES.length / 2))
```

Comment:
nice fix, but it's probably even easier to understand by assigning to variable first:
```
const indexOfHalf = Math.ceil(ACTIVE_PARTIES.length / 2)
const first = ACTIVE_PARTIES.slice(0, indexOfHalf)
const second = ACTIVE_PARTIES.slice(indexOfHalf)
```